docs/dashboard-spec.md

```diff
@@ -14,6 +14,31 @@ packs/<pack_name>/dashboards/apache.json
 
 The schema adheres to the `DashboardInput` used in the [dashboardCreate](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/core-concepts/dashboards-api-migration-insights-api-nerdgraph/#dashboard-create) mutation in NerdGraph.
 
+
+```yaml
+####################################################################################
+# Metadata
+####################################################################################
+
+# the name of the Dashboard
+name: string, required
+
+# A short description for this Dashboard
+description: string, required
+
+# List of screenshot URLs related to this Dashboard
```

Comment:
This comment needs to be updated

Reply:
how so? What's the update?

Reply:
That comment refers to a list of URLs, but it's describing the `description` field